### ISSUE TITLE
[BUG] - New install yields console warning: Prop `aria-label` did not match.

### ISSUE THREAD

**Issue by CitizenBeta** [nextui-org/nextui#1699]
### NextUI Version

2.1.13

### Describe the bug

If you create a new app using this:

`npx create-next-app -e https://github.com/nextui-org/next-app-template`

An run `npm run dev`

If you open the browser and the console you get hit with a fat warning:

> app-index.js:31 Warning: Prop `aria-label` did not match. Server: "Switch to light mode" Client: "Switch to dark mode"
>     at label
>     at ThemeSwitch (webpack-internal:///(app-pages-browser)/./components/theme-switch.tsx:21:11)
>     at li
>     at eval (webpack-internal:///(app-pages-browser)/./node_modules/@nextui-org/navbar/dist/chunk-MG5LCNUN.mjs:21:11)
>     at ul
>     at eval (webpack-internal:///(app-pages-browser)/./node_modules/@nextui-org/navbar/dist/chunk-PSG7VTZC.mjs:20:11)
>     at header
>     at nav
>     at eval (webpack-internal:///(app-pages-browser)/./node_modules/@nextui-org/navbar/dist/chunk-PZVBZM3A.mjs:27:11)
>     at div
>     at f (webpack-internal:///(app-pages-browser)/./node_modules/next-themes/dist/index.module.js:8:597)
>     at $ (webpack-internal:///(app-pages-browser)/./node_modules/next-themes/dist/index.module.js:8:348)
>     at div
>     at $f57aed4a881a3485$var$OverlayContainerDOM (webpack-internal:///(app-pages-browser)/./node_modules/@react-aria/overlays/dist/import.mjs:911:55)
>     at $f57aed4a881a3485$export$178405afcd8c5eb (webpack-internal:///(app-pages-browser)/./node_modules/@react-aria/overlays/dist/import.mjs:878:21)
>     at $f57aed4a881a3485$export$bf688221f59024e5
>     at $18f2051aff69b9bf$export$a54013f0d02a8f82 (webpack-internal:///(app-pages-browser)/./node_modules/@react-aria/i18n/dist/real-module.mjs:163:19)
>     at NextUIProvider (webpack-internal:///(app-pages-browser)/./node_modules/@nextui-org/system/dist/chunk-5DCY5R7Z.mjs:13:3)
>     at Providers (webpack-internal:///(app-pages-browser)/./app/providers.tsx:15:11)
>     at body
>     at SessionProvider (webpack-internal:///(app-pages-browser)/./node_modules/next-auth/react/index.js:455:24)
>     at AuthProvider (webpack-internal:///(app-pages-browser)/./components/auth/auth-provider.tsx:11:11)
>     at html
>     at RedirectErrorBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/redirect-boundary.js:73:9)
>     at RedirectBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/redirect-boundary.js:81:11)
>     at NotFoundErrorBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/not-found-boundary.js:54:9)
>     at NotFoundBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/not-found-boundary.js:62:11)
>     at DevRootNotFoundBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/dev-root-not-found-boundary.js:32:11)
>     at ReactDevOverlay (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/react-dev-overlay/internal/ReactDevOverlay.js:66:9)
>     at HotReload (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/react-dev-overlay/hot-reloader-client.js:326:11)
>     at Router (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/app-router.js:147:11)
>     at ErrorBoundaryHandler (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/error-boundary.js:82:9)
>     at ErrorBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/error-boundary.js:110:11)
>     at AppRouter (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/app-router.js:417:13)
>     at ServerRoot (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/app-index.js:123:11)
>     at RSCComponent
>     at Root (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/app-index.js:139:11)

### Steps to Reproduce the Bug or Issue

1. Create a new app using NextUI
2. Change the light/dark theme
3. Refresh the browser
4. Check the console log

### Expected behavior

Not displaying a console warning

### Operating System Version

Linux

### Browser

Chrome

**Comment by ekopras18** [comment]
bug Fixed in NextUI Version 2.2.9 

add this -> 'isSSR'

`useSwitch({ isSelected: theme === "light" || isSSR, "aria-label": Switch to ${theme === "light" || isSSR ? "dark" : "light"} mode,onChange,});`

**Comment by CitizenBeta** [comment]
> bug Fixed in NextUI Version 2.2.9
> 
> add this -> 'isSSR'
> 
> `useSwitch({ isSelected: theme === "light" || isSSR, "aria-label": Switch to ${theme === "light" || isSSR ? "dark" : "light"} mode,onChange,});`

This fixed it for me. The copy and paste code if anyone else needs it is:

```
const {
        Component,
        slots,
        isSelected,
        getBaseProps,
        getInputProps,
        getWrapperProps,
    } = useSwitch({
        isSelected: theme === "light" || isSSR,
        "aria-label": "Switch to " + (theme === "light" || isSSR ? "dark" : "light") + "mode",
        onChange
    });
```
